_src/recipes/013-add-a-pagination.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Add a Pagination  
+authorName: Akbarsait
+authorUrl: https://www.akbarsait.com
+---
+
+**Pagination** helps in displaying the blog post or contents organized in your blog or website. 11ty brings the pagination handy by has it built into collections itself. 
+
+### Prerequisites
+
+- [Add a layout](../add-a-layout/)
+- [Create a Collection](../create-a-collection)
+
+### Directions
+
+1. Follow the instructions on [Create a Collection](../create-a-collection) and by having a tag with name as *general* to each entry of the blog post, we have the collection established. Now add the following code to our **index.md** file. 
+
+
+```md
+---
+layout: layout
+title: My Web Blog
+pagination:
+  data: collections.general
+  size: 2
+  alias: posts
+---
+```
+
+2. In the above code, we have added the **Pagination** key to our template's front matter and provided our collection dataset from blog post to **data** key and to limit the number of posts displayed in a page can be controlled using the **size** key and used **alias** name for collection. Now use the following lines of code to your **index.md** page for creating the pagination navigation. 
+
+```md
+## My Blog Posts
+
+{%- for post in posts %}
```

Comment:
Use `{% raw %}{% endraw %}` here to ensure the Nunjucks doesn't compile inside of the code block.